fe1-web/components/__tests__/ChirpCard.test.tsx

```diff
@@ -8,10 +8,15 @@ const chirp = new Chirp({
   id: new Hash('1234'),
   text: 'Don\'t panic.',
   sender: new PublicKey('Douglas Adams'),
-  time: new Timestamp(1609455600),
+  time: new Timestamp(1609455600), // 31 December 2020
   likes: 42,
 });
 
+beforeAll(() => {
+  jest.useFakeTimers('modern');
+  jest.setSystemTime(new Date(1620255600000)); // 5 May 2021
```

Comment:
would it make sense to update the `Timestamp` mock as well, so that they provide the same absolute time? Or even replace that mock with this fake timer everywhere? 😅

Reply:
I think this is cleaner, in the way that the system time is set directly. I think I'll replace that mock with the timer.